arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/CreateSection.java

```diff
@@ -62,11 +62,15 @@
 
 public class CreateSection extends TraceSection
     implements PostOpcodeDefer,
-        ImmediateContextEntryDefer,
+        ContextEntryDefer,
         PostRollbackDefer,
         ContextReEntryDefer,
         PostTransactionDefer {
 
+  private Address creatorAddress;
```

Comment:
final

Reply:
We can make it final, but currently we only initialize those values once the CREATE is unexceptional.